### PR TITLE
feature/785

### LINKED ISSUE #785: New Airport: KPIT (Pittsburgh Int'l)

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves #785 

The purpose of this pull request is to figure how git works and add a new airport in the meantime

### Completion Checklist
- [x] terrain file
- [x] polygonal airspace boundary
- [x] all SIDs
- [x] all STARs
- [x] information to validate traffic levels
- [x] base video map
- [x] no console errors or warnings
- [x] airport guide
---

According to Wikipedia, KPIT has 144,563 yearly operations. Traffic level should fall between 16.5acph and 49.5acph. Currently, there's a total of 16.6acph 👍

### CODE REVIEW

**Comment by erikquinn** [comment]
Extra points if you make the PIT2 pronounced `Shitsburgh Two" :wink:

**Comment by Fechulo** [comment]
All done! Ready for review

**Comment by Fechulo** [comment]
Needs a guide, so `WIP` it is

File: assets/airports/kpit.json [hunk 1]
```diff
@@ -0,0 +1,4316 @@
...
+                ["YNG*", "CUTTA"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "KPIT",
+            "destination": "",
+            "category": "departure",
+            "route": "KPIT.PIT2.AHTIY..BURGS",
```
Comment (erikquinn) [inline assets/airports/kpit.json]: These and all spawn patterns will need to include runways, such that the entry/exit matches that which is specified in the procedure definition. This is true in spawn patterns and any time being issued a reroute. It has to do with the way routes are interpreted.

**Comment by Fechulo** [comment]
Should be ready to go now! 😄

**Pull request merged**